WooCommerce/Classes/Tools/Developer/DeveloperEmailChecker.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+struct DeveloperEmailChecker {
+    private static let developerEmails = ["@automattic.com", "@a8c.com"]
+
+    /// Checks if an email belongs to app developer (Automattic).
+    ///
+    static func isDeveloperEmail(email: String) -> Bool {
+        return (developerEmails.first { email.contains($0) }) != nil
```

Comment:
I haven't actually tested this, but should this work as well?

```
return developerEmails.contains(email)
```

I find that slightly easier to parse (of course, this is a personal preference, feel free to ignore it)

Reply:
Aaaaaand I just realised that my comment does not make sense, as we are checking the domain only 🤦‍♂ 🤷‍♂